src/requirements.in

```diff
@@ -6,6 +6,7 @@ django-filter == 2.2.0
 django-healthchecks == 1.4.2
 django-postgres-unlimited-varchar == 1.1.0
 django-gisserver == 1.1.1
+django-vectortiles == 0.1.0
```

Comment:
evt `django-vectorriles[mapbox]` als requirement?

Reply:
Volgens de README krijgen we dan geen PostGIS support. Die hebben we wel nodig.